File: src/PHP/CreationalPatterns/AbstractFactory/Furniture/Products/ContemporaryChair.php
<?php

namespace CreationalPatterns\AbstractFactory\Furniture\Products;

use CreationalPatterns\AbstractFactory\Furniture\Interfaces\IFurniture;

class ContemporaryChair implements IFurniture {
    public function showFurnitureStyle(): void
    {
        echo "I am a contemporary chair\n";
    }
}

File: src/C#/CreationalPatterns/AbstractFactory/Furniture/Products/ContemporaryChair.cs
using Furniture.Interfaces;
using System;
using System.Collections.Generic;
using System.Text;

namespace Furniture.Products
{
    public class ContemporaryChair : IFurniture
    {
        public void ShowFurnitureStyle()
        {
            Console.WriteLine("I am a contemporary chair");
        }
    }
}
